### PR TITLE
update: Added first version of RSS plugin

### PR DESCRIPTION
**Pull request by mrshu**
* Added first version of RSS plugin which periodically looks for updates
  in given RSS feeds and prints them to all channels to which its bot is
  connected.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

**Comment by mrshu** [comment]
to @Adman, @pepol

**Comment by mrshu** [comment]
Per our  discussion with @adman I added link shortening.

**Comment by Adman** [comment]
@mrshu I would suggest shortening the url only if its length is more than XY characters

**Comment by mrshu** [comment]
@Adman that's a very good point. Would something like 50 characters work? Or maybe just 30?

Also, I am wondering though whether it (the fact that we are in fact shortening URLs) should be configurable.

**Comment by mrshu** [comment]
@Adman allright, I've made it configurable and set the length to 50. It worked pretty well in my tests.

**Comment by Adman** [comment]
@mrshu isn't 50 too many?

**Comment by mrshu** [comment]
@Adman considering that `http://` combined with the final slash is already 8 characters I do not think so.

Also, let me note here that given the amount of variables that are configurable for this plugin, and the fact that there is no effective way of writing documentation for it, it would make way more sense to have each plugin as a separate Python module. That way, we could easily have configuration and the installation/development of respective plugins would be way easier too.

The only thing that is missing is their central repository. Looking around, we can see that this is the model hubot uses (https://github.com/hubot-scripts). So what we might do would be to have these plugins under one organization (to which all of the collaborators to this repository would have the same rights as they currently have to this repository) and also we would tag these packages with a special `brutal-plugin` tag or something like that. That way, you could easily search for these packages on pypi.

@Adman @JakubNvk @pepol thoughts?

**Comment by Adman** [comment]
lgtm

File: rss.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                self.storage[feed] = ''
+        self.loop_task(self.update_interval, self.check_feeds, now=False)
+
+    def check_feeds(self):
+        for feed in self.feeds:
+            d = feedparser.parse(feed)
+            last_id = self.storage[feed]
+            first_id = None
+
+            i = 1
```
Comment (Adman) [inline rss.py]: @mrshu You could probably use enumerate rather than a pure variable

Comment (mrshu) [inline rss.py]: @Adman good point, will refactor

**Comment by mrshu** [comment]
@Adman I refactored what you asked for, feel free to punch the big green button.

**Pull request merged**